prow/plugins/approve/plugins.yaml

```diff
@@ -0,0 +1,27 @@
+# Approve plugin example `plugins.yaml`
```

Comment:
Calling this `plugins.yaml` might be confusing (by which I mean it confused me) - perhaps `example.yaml` or something might be clearer?

Reply:
Ok, I see what you mean (i.e. since `plugins.yaml` already has meaning in the context of prow).    I will update the naming to `example.yaml`.